### PR TITLE
Created a seed script to generate 9 example users

### LINKED ISSUE #10: Write scripts to seed the db using Prisma
Resource to look at: https://www.prisma.io/blog/nestjs-prisma-rest-api-7D056s1BmOL0

### PR DESCRIPTION
**Pull request by Suraj-Ram**
The example data is still pending, as we need to write scripts to generate sample data for the rest of the tables.

### CODE REVIEW

File: src/testData/makeExamples.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+const { PrismaClient } = require('@prisma/client')
```
Comment (daisykucharski) [inline src/testData/makeExamples.js]: Is this file still necessary now that we're using the prisma seeding stuff? If it isn't we should just delete it

Comment (Suraj-Ram) [reply src/testData/makeExamples.js]: Yeah that isn't needed

**Review by daisykucharski — APPROVED** [review]
LGTM 🚀
